alvr/server_openvr/src/lib.rs

```diff
@@ -382,16 +382,24 @@ extern "C" fn wait_for_vsync() {
         .video
         .optimize_game_render_latency
     {
-        SERVER_CORE_CONTEXT
-            .read()
-            .as_ref()
-            .and_then(|ctx| ctx.duration_until_next_vsync())
+        // We default to 10ms in the event that a headset hasn't connected,
+        // because otherwise systems can lock up with 100% GPU.
+        Some(
+            SERVER_CORE_CONTEXT
+                .read()
+                .as_ref()
+                .and_then(|ctx| ctx.duration_until_next_vsync())
+                .unwrap_or(Duration::from_millis(10)),
```

Comment:
I don't think it's needed to fallback to a specific duration also here

Reply:
This one only happens with `optimize_game_render_latency` enabled  (ie, a proper vsync interval) but with no stats manager created yet because a headset hasn't connected